cob_bringup/robots/cob4-1.launch

```diff
@@ -5,17 +5,17 @@
 	<arg name="pkg_env_config" default="$(find cob_default_env_config)"/>
```

Comment:
Good point, this is something more general, though.
The point of commenting out the `upload_navigation_goals.launch` below was copied from cob4-2.
By doing so, indeed the launch args become obsolete (for all robots)!

But if we uncomment the `upload_navigation_goals.launch` here, where else is it loaded?
Should be somewhere within navigation, right?
We just should make sure that navigation does not rely on navigation goals to be uploaded via robot.launch but take care about it itself....
@ipa320/navigation-push do you have a comment here?